frontend_v2/src/app/components/challenge/challengeanalytics/challengeanalytics.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="challenge-card">
+  <div class="ev-card-panel card-bt-margin">
+    <div class="ev-md-container ev-panel-title">
+      <div class="row row-lr-margin">
+        <div class="col-sm-12 col-xs-12 col-lr-pad">
+          <h5 class="fw-light">Challenges Analytics</h5>
+        </div>
+        <div class="col-sm-12 md-body-1 col-lr-pad fs-16 fw-light">
+          Here you get the complete analysis of <b>{{ challenge['title'] }}</b
+          >. This includes getting total number submissions, total number of participated teams, number of phases,
+          number of submissions in each phases etc.
+        </div>
+      </div>
+    </div>
+    <div class="ev-md-container">
+      <div class="analytics-details">
+        <div class="row">
+          <div class="col s12 m4 l4">
+            <div class="image-container analytics-image">
+              <img *ngIf="challenge['image'] !== null" [src]="challenge['image']" />
+              <img *ngIf="challenge['image'] == null" src="assets/images/404.png" />
+            </div>
+          </div>
+          <div class="col m8 l8">
+            <div class="card-content analytics-card-content">
+              <h5 class="fw-light">
+                {{ challenge['title'] }}
+              </h5>
+              <span class="text-light-black fs-16 fw-light">Hosted by: </span>
+              {{ challenge['creator']['team_name'] }} <br />
+              <span class="main-title fs-16 fw-light">
+                Total Participant Teams:
+              </span>
+              <span class="show-count fs-16 fw-light">
+                {{ totalChallengeTeams }}
+              </span>
+              <br />
```

Comment:
Not sure if this file is indented properly. In some other instances, arrow brackets are spanning across multiple lines.

Reply:
@lunayach the file indentation is now taken care you the pre-commit hook `preetier`. So even if I try to change the indentation it won't be affected here due to the pre-commit hook.

Reply:
Ahh, I see. Then it's weird that per-commit hook is enforcing indentations like this. You could leave it like this for now.